crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -63,21 +63,21 @@ def upgrade!
       end
     end
 
-    def maintenance_updates_installed?
-      Open3.popen3("zypper patch-check") do |_stdin, _stdout, _stderr, wait_thr|
-        case wait_thr.value.exitstatus
-        when 100
-          Rails.logger.warn(
-            "ZYPPER_EXIT_INF_UPDATE_NEEDED: patches available for installation."
-          )
-          false
-        when 101
-          Rails.logger.warn(
-            "ZYPPER_EXIT_INF_SEC_UPDATE_NEEDED: security patches available for installation."
-          )
-          false
-        else
-          true
+    def maintenance_updates_status
+      {}.tap do |ret|
+        Open3.popen3("zypper patch-check") do |_stdin, _stdout, _stderr, wait_thr|
```

Comment:
Don't we also need to check that the update repo metadata is up to date?  Or does `zypper patch-check` always automatically refresh the update repos?